bin/kafka-producer-perf-test.sh

```diff
@@ -17,4 +17,4 @@
 if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
-exec $(dirname $0)/kafka-run-class.sh kafka.tools.ProducerPerformance $@
+exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.clients.tools.ProducerPerformance $@
```

Comment:
@omkreddy : Discussed this with @granders offline. It seems there is no strong benefit of argparse4 over jopt-simple. For consistency, we can change VerifiableProducer to use jopt-simple for now. We just want to make sure that existing system tests don't break after the change.